sdk_contrib/aws_v1_support/src/test/java/io/opentelemetry/sdk/contrib/trace/aws/Ec2ResourceTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.sdk.contrib.trace.aws;
+
+import static org.junit.Assert.assertEquals;
+
+import io.opentelemetry.sdk.resources.Resource;
+import org.junit.Test;
+
+public class Ec2ResourceTest {
+
+  @Test
+  public void shouldReturnResourceWithOnlyCloudProviderLabelIfNotRunningOnEc2() {
+    Resource resource = Ec2Resource.getResource();
+    assertEquals(1, resource.getLabels().size());
```

Comment:
can we add an assertion about the label key that was supplied?

Reply:
Based on one of my comments I think tests will be improved by the proposed refactoring.

Reply:
Done